docs/home-manager.adoc

```diff
@@ -42,3 +42,36 @@ Then we should be able to use the given module. E.g.
   };
 }
 ----
+
+=== Custom vim plugins
+
+It's possible to add custom vim plugins by using the startPlugins and lua DAG settings. First we install the plugin by adding it to startPlugins, I'll be using the nvim-surround plugin for example. This process should be very similar for nixos installations.
```

Comment:
Please don't use first person perspective in the documentation.

A suggestion would be 

"This example uses nvim-surround"